src/gui/DatabaseWidget.h

```diff
@@ -24,6 +24,10 @@
 #include <QStackedWidget>
 #include <QTimer>
 
+#if defined(__linux) || defined(__linux__) || defined(linux)
```

Comment:
See above. Is this header available on other *nix platforms? If not, are there alternatives?